apisix/control/v1.lua

```diff
@@ -156,6 +156,40 @@ function _M.get_health_checker()
     return 200, info
 end
 
+local function iter_add_get_routes_info(infos, values, route_id)
```

Comment:
The infos can be created inside this function, and there is no need to pass out a flag -- use nil if route_id given but not matched.